Tools/Frame_params/Solo_AC35.param

```diff
@@ -1,172 +1,215 @@
-SERIAL4_BAUD,230
-SERIAL4_PROTOCOL,1
-SERIAL5_BAUD,57
-SERIAL5_PROTOCOL,-1
-SR0_ADSB,5
-SR0_EXT_STAT,0
-SR0_EXTRA1,0
-SR0_EXTRA2,0
-SR0_EXTRA3,0
-SR0_PARAMS,0
-SR0_POSITION,0
-SR0_RAW_CTRL,0
-SR0_RAW_SENS,10
-SR0_RC_CHAN,0
-SR1_ADSB,5
-SR1_EXT_STAT,2
-SR1_EXTRA1,5
-SR1_EXTRA2,5
-SR1_EXTRA3,1
-SR1_PARAMS,10
-SR1_POSITION,1
-SR1_RAW_CTRL,3
-SR1_RAW_SENS,1
-SR1_RC_CHAN,1
-SR2_ADSB,5
-SR2_EXT_STAT,0
-SR2_EXTRA1,0
-SR2_EXTRA2,0
-SR2_EXTRA3,0
-SR2_PARAMS,0
-SR2_POSITION,0
-SR2_RAW_CTRL,0
-SR2_RAW_SENS,0
-SR2_RC_CHAN,0
-SR3_ADSB,5
-SR3_EXT_STAT,0
-SR3_EXTRA1,0
-SR3_EXTRA2,0
-SR3_EXTRA3,0
-SR3_PARAMS,0
-SR3_POSITION,0
-SR3_RAW_CTRL,0
-SR3_RAW_SENS,0
-SR3_RC_CHAN,0
-THR_DZ,10
-VEL_XY_FILT_HZ,5
-VEL_XY_I,0.5
-VEL_XY_IMAX,1000
-VEL_XY_P,1.4
-VEL_Z_P,5
-WP_YAW_BEHAVIOR,2
-WPNAV_ACCEL,340
-WPNAV_ACCEL_Z,160
-WPNAV_LOIT_JERK,1400
-WPNAV_LOIT_MAXA,321
-WPNAV_LOIT_MINA,138
-WPNAV_LOIT_SPEED,700
-WPNAV_RADIUS,200
-WPNAV_SPEED,1100
-WPNAV_SPEED_DN,240
-WPNAV_SPEED_UP,320
+#NOTE: 4/13/2017: All required 3DR Solo parameters for ArduCopter 3.5±.
+ACCEL_Z_D,0
+ACCEL_Z_FF,0
+ACCEL_Z_FILT,20
+ACCEL_Z_I,1.5
+ACCEL_Z_IMAX,800
+ACCEL_Z_P,0.75
+ACRO_BAL_PITCH,1
+ACRO_BAL_ROLL,1
+ACRO_RP_EXPO,0.3
+ACRO_RP_P,4
+ACRO_THR_MID,0.4
+ACRO_TRAINER,2
+ACRO_Y_EXPO,0
+ACRO_YAW_P,1
+ADSB_ENABLE,0
+AHRS_COMP_BETA,0.1
+AHRS_EKF_TYPE,2
+AHRS_GPS_GAIN,1
+AHRS_GPS_MINSATS,6
+AHRS_GPS_USE,1
+AHRS_ORIENTATION,12
+AHRS_RP_P,0.2
+AHRS_WIND_MAX,0
+AHRS_YAW_P,0.2
+ANGLE_MAX,2000
+ATC_ACCEL_P_MAX,36000
+ATC_ACCEL_R_MAX,36000
+ATC_ACCEL_Y_MAX,7000
+ATC_ANG_LIM_TC,1
+ATC_ANG_PIT_P,4.5
+ATC_ANG_RLL_P,4.5
+ATC_ANG_YAW_P,3.42
+ATC_ANGLE_BOOST,1
+ATC_RAT_PIT_D,0.0036
+ATC_RAT_PIT_FF,0
+ATC_RAT_PIT_FILT,20
+ATC_RAT_PIT_I,0.09
+ATC_RAT_PIT_IMAX,0.444
+ATC_RAT_PIT_P,0.135
+ATC_RAT_RLL_D,0.0036
+ATC_RAT_RLL_FF,0
+ATC_RAT_RLL_FILT,20
+ATC_RAT_RLL_I,0.09
+ATC_RAT_RLL_IMAX,0.444
+ATC_RAT_RLL_P,0.135
+ATC_RAT_YAW_D,0
+ATC_RAT_YAW_FF,0
+ATC_RAT_YAW_FILT,3.151
+ATC_RAT_YAW_I,0.092
+ATC_RAT_YAW_IMAX,0.1
+ATC_RAT_YAW_P,0.92
+ATC_RATE_FF_ENAB,1
+ATC_SLEW_YAW,1000
+ATC_THR_MIX_MAN,0.5
+ATC_THR_MIX_MAX,0.5
+ATC_THR_MIX_MIN,0.1
+AVOID_ENABLE,3
+BATT_CAPACITY,5200
+BATT_MONITOR,5
+BRD_IMU_TARGTEMP,60
+BRD_PWM_COUNT,4
+BRD_SAFETY_MASK,16368
+BRD_SAFETYENABLE,0
+BRD_SER1_RTSCTS,1
+BRD_TYPE,3
+BTN_ENABLE,0
+CAM_DURATION,10
+CAM_TRIGG_TYPE,0
+COMPASS_AUTODEC,1
+COMPASS_CAL_FIT,16
+COMPASS_EXTERN2,0
+COMPASS_EXTERN3,0
+COMPASS_EXTERNAL,1
+COMPASS_LEARN,0
+COMPASS_ORIENT,38
+COMPASS_ORIENT2,0
+COMPASS_ORIENT3,0
+COMPASS_PRIMARY,0
+COMPASS_USE,1
+COMPASS_USE2,1
+COMPASS_USE3,0
+DISARM_DELAY,5
+EK2_ENABLE,1
+EK2_GPS_TYPE,0
+EK3_ENABLE,0
+FENCE_ACTION,0
+FENCE_ALT_MAX,122
+FENCE_ENABLE,1
+FENCE_MARGIN,2
+FENCE_RADIUS,600
+FENCE_TOTAL,0
+FENCE_TYPE,1
+FLTMODE1,5
+FLTMODE2,5
+FLTMODE3,5
+FLTMODE4,5
+FLTMODE5,5
+FLTMODE6,5
+FRAME_CLASS,1
+FRAME_TYPE,1
+FS_BATT_ENABLE,2
+FS_BATT_MAH,520
+FS_BATT_VOLTAGE,13.9
+FS_EKF_ACTION,2
+FS_THR_ENABLE,1
+FS_THR_VALUE,950
+GND_EFFECT_COMP,1
+GPS_HDOP_GOOD,200
+GPS_NAVFILTER,6
```

Comment:
@WickedShell I used auto so it will work with whatever someone hooks up.  Is there anything other than ublox these days?  Does HERE use ublox?  Is there a reliability risk when using Auto? I'm not heart set on either value, was just aiming for flexibility.

As for the other settings, that's Solo specific.